sdk/confidentialledger/Azure.Security.CodeTransparency/src/CodeTransparencyClient.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Security.CodeTransparency
+{
+    /// <summary>
```

Comment:
This comment will be used for public docs on MS Learn.   I don't think we'd want this as the type summary.    

I didn't do a deep pass over all doc comments, but that may be a good idea prior to merging.  All comments on public members are published to the official MS Docs API pages.

Reply:
Cleaned up a bit